mobile/src/main/java/com/google/samples/apps/iosched/ui/sessiondetail/SessionDetailActivity.kt

```diff
@@ -89,8 +100,9 @@ class SessionDetailActivity : AppCompatActivity() {
     }
 
     private fun getSessionId(intent: Intent): String? {
-        return intent.data?.getQueryParameter(QUERY_SESSION_ID) // for iosched://sessions/{id}
-                ?: intent.getStringExtra(EXTRA_SESSION_ID)
+        return intent.data?.lastPathSegment // for iosched://sessions/{id}
```

Comment:
Will `.lastPathSegment` capture `sessions` if the uri is of the form `iosched://sessions?session_id={id}`?

In any case this might be better as a separate change if there's a bug with the current code here.

Reply:
Good point! I'll revert this piece. I am not sure if there is a bug because alarm manager seems to send the intent with query params but I could not get the query params to work through adb so there is some magic happening somewhere. Let's assume it is working but if it is not, then we can create a bug and fix it outside of the scope of this change.